krun/platform.py

```diff
@@ -1036,9 +1029,22 @@ def _check_tickless_kernel(self):
 
         debug("Checking linux kernel is tickless")
 
+        if self.num_cpus == 1:
+            # Tickless mode doesn't make sense on a single-cpu system, as you
+            # need at least on non-adaptive ticks CPU to do various
```

Comment:
"at least on non-adaptive" doesn't parse. [I can guess the overall meaning, but only because I know the area!]

Reply:
ah, `on` -> `one`. Good catch.

Reply:
Even then it's still a bit hard to parse. Maybe "non-adaptive-ticks" helps?